datalad/support/gitrepo.py

```diff
@@ -1897,19 +1956,19 @@ def _maybe_open_ssh_connection(self, remote, prefer_push=True):
             if url and is_ssh(url):
                 ssh_manager.get_connection(url).open()
 
-    def update_remote(self, name=None, verbose=False):
+    def update_remote(self, name: Optional[str] = None, verbose: bool = False) -> None:
         """
         """
         options = ["-v"] if verbose else []
         self._maybe_open_ssh_connection(name)
-        name = [name] if name else []
+        namelst = [name] if name else []
         self.call_git(
-            ['remote'] + name + ['update'] + options,
+            ['remote'] + namelst + ['update'] + options,
             expect_stderr=True
         )
 
-    def fetch(self, remote=None, refspec=None, all_=False, git_options=None,
-              **kwargs):
+    def fetch(self, remote: Optional[str] = None, refspec: str | list[str] | None = None, all_: bool = False, git_options: Optional[list[str]] = None,
```

Comment:
when are we to make it ` ... | None = None` vs ` Optional[...] = None` ?

Reply:
oh, ok. Might be better to stick to one or another since otherwise it makes some folks like me wonder on how those two might differ ;)